MozziGuts_impl_MBED.hpp

```diff
@@ -166,7 +166,13 @@ void stopMozzi() {
 mbed::Ticker audio_output_timer;
 
 #if (EXTERNAL_AUDIO_OUTPUT == true)
-#define AUDIO_OUTPUT_OUTSIDE_CALLBACK true
+volatile bool audio_output_requested = false;
+inline void defaultAudioOutputCallback() {
+  audio_output_requested = true;
+}
+#define AUDIO_HOOK_HOOK { if (audio_output_requested) { audio_output_requested = false; defaultAudioOutput(); } }
+#else
+#define defaultAudioOutputCallback defaultAudioOutput
```

Comment:
Note that the latest fixes there are not yet merged: https://github.com/arduino-libraries/Arduino_AdvancedAnalog/pull/36 . Are you sure you are using that?

Reply:
Copy-and-paste fail?

Reply:
Oups, and an epic one! I wanted to ask if this was for forward compatibility for other output modes as `defaultAudioOutput` is never used (yet?) in this implementation.

Reply:
You're right. It doesn't really make sense as long as there is no TIMED_PWM. I'll clean up (this and other bits) later.